addons/l10n_ec/data/template/account.group-ec.csv

```diff
@@ -9,6 +9,7 @@
 "ec1102","1102","Financial assets","ec11","Activos financieros"
 "ec110205","110205","Unrelated notes and accounts receivable","ec1102","Documentos y cuentas por cobrar cliente no relacionados"
 "ec11020501","11020501","Non-interest bearing notes and accounts receivable from unrelated ordinary activities interest-bearing","ec110205","Documentos y cuentas por cobrar cliente no relacionados de actividades ordinarias que no generen intereses"
+"ec11020502","11020502","Customer checks","11020502","Cheques de clientes"
```

Comment:
@stevTresCloud Something is off with the parent_id here. I guess that is why the staging failed.

Reply:
@jco-odoo Hi thanks!. I fixed it
You can try again please.